substrate/bin/utils/chain-spec-builder/src/lib.rs

```diff
@@ -146,7 +153,7 @@ pub enum ChainSpecBuilderCmd {
 	AddCodeSubstitute(AddCodeSubstituteCmd),
 }
 
-/// Create a new chain spec by interacting with the provided runtime wasm blob.
+/// Create a new solo chain spec by interacting with the provided runtime wasm blob.
```

Comment:
```suggestion
/// Create a new chain spec by interacting with the provided runtime wasm blob.
```
I mean we now support parachains :D